src/main/webapp/app/app.component.html

```diff
@@ -8,8 +8,14 @@
     </div>
   </mat-toolbar>
   <section class="content">
+    
```

Comment:
Why are you adding the new lines to the html?

Reply:
By new line means the new code then it is for the search feature that I have Implemented.